setup.py

```diff
@@ -208,7 +208,7 @@ def op_enabled(op_name):
     git_branch = "unknown"
 
 
-def create_dir_symlink(src, dest):
+def create_dir_symlink(dest,src):
```

Comment:
Perhaps keep these parameters in this order since that matches the os.symlink(src, dest) convention?  And switch the paths below where this is used?

Reply:
I did your sugestion here (commit 1f5384f6f3bf2ed40add713d073c28d6ad018ee7) @loadams.